sublime-package.json

```diff
@@ -26,6 +26,41 @@
               "type": "string",
               "deprecationMessage": "Use the \"selector\" key instead."
             },
+            "FileWatcher": {
+              "type": "object",
+              "examples": [
+                {
+                  "glob": "{**/*.js,**/*.ts,**/*.json}",
+                  "kind": ["create", "change", "delete"],
```

Comment:
Maybe this should be called `type`...
Spec uses `kind` when registering but `type` when triggering events.